### PR TITLE
Fix broken urls in flutter-for docs

### PR DESCRIPTION
**Pull request by saintmalik**
## Summary 

Easy reading and easy navigation for others

### CODE REVIEW

File: src/docs/cookbook/persistence/sqlite.md [hunk 1]
```diff
@@ -476,9 +476,9 @@ class Dog {
 ```
 
 
-[`delete()`]: {{site.pub-api}}/sqflite/latest/sqlite_api/DatabaseExecutor/delete.html'
-[`insert()`]: {{site.pub-api}}/sqflite/latest/sqlite_api/DatabaseExecutor/insert.html
+[`delete()`]: {{site.pub-api}}/sqflite_common/latest/sqlite_api/DatabaseExecutor/delete.html'
```
Comment (sfshaza2) [inline src/docs/cookbook/persistence/sqlite.md:479]: Overall, this PR looks good, but that single quite at the end of this line should be deleted.

Comment (saintmalik) [reply src/docs/cookbook/persistence/sqlite.md:479]: Ohh yeah, I missed that, Thanks, I will push the fix now

Comment (saintmalik) [reply src/docs/cookbook/persistence/sqlite.md]: I pushed the correction fix already, Thanks

**Review by lucalves — APPROVED** [review]

**Review by sfshaza2 — APPROVED** [review]
Thanks for this, @saintmalik! It looks good! LGTM

**Pull request merged**